Add Zoho Corporation

Anon-sec opened the pull request:
**Type of pull request:** product addition

**Related issues:** list the related issues here (if applicable)

---
Added privacy statistics for Zoho Corporation. For help, see our contribution guide: http://privacyspy.org/contribute/

Anon-sec committed:
Add files via upload

Anon-sec committed:
Add files via upload

Anon-sec committed:
Update CONTRIBUTORS.toml

doamatto commented on icons/zoho.jpg:
This needs to be a square icon

doamatto commented on products/zoho.toml:
```diff
@@ -0,0 +1,77 @@
+
+
+
+
```
Lot of blank space here that shouldn't be.

doamatto commented on products/zoho.toml:
```diff
@@ -0,0 +1,77 @@
+
+[rubric.third-party-collection]
+value = "no"
+
```
Needs citation.

gg-sys1 replied:
added citation

doamatto commented on products/zoho.toml:
```diff
@@ -0,0 +1,77 @@
+
+[rubric.third-party-access]
+value = "yes-unspecified-critical"
+
```
Needs citation.

gg-sys1 replied:
added citation

doamatto commented on products/zoho.toml:
```diff
@@ -0,0 +1,77 @@
+
+[rubric.security]
+value = "yes"
+
```
Needs citation.

gg-sys1 replied:
added citation

doamatto commented on products/zoho.toml:
```diff
@@ -0,0 +1,77 @@
+
+[rubric.noncritical-purposes]
+value = "opt-out-some"
+
```
Needs citation.

doamatto commented on products/zoho.toml:
```diff
@@ -0,0 +1,77 @@
+
+[rubric.list-collected]
+value = "generally"
+
```
Needs citation.

doamatto commented on products/zoho.toml:
```diff
@@ -0,0 +1,77 @@
+[rubric.data-breaches]
+value = "eventually"
+notes = [
+  "The policy does not specify a strict timeframe for notifying users about breaches but mentions compliance with laws and regulations, suggesting eventual notification."
```
We play devil's advocate and assume that although the law says they will, the policy needs to affirm it (even in the case of the GDPR's 72-hour rule). No mention would mean `no` here.

doamatto commented on products/zoho.toml:
```diff
@@ -0,0 +1,77 @@
+[rubric.revision-notify]
+value = "yes"
+notes = [
+  "Zoho mentions that they will notify users of significant changes to the privacy policy either through an email or a prominent notice on the website, giving at least 30 days' advance notice."
```
Citation is more beneficial than a note.

gg-sys1 replied:
Added Citation as well.

doamatto commented on products/zoho.toml:
```diff
@@ -0,0 +1,77 @@
+[rubric.history]
+value = "last-modified"
+citations = [
+  "We may modify the Privacy Policy at any time, upon notifying you through a service announcement or by sending an email to your primary email address"
```
It's more beneficial to cite the last updated time rather than a quote better fit for `revision-notify`

Anon-sec replied:
Is there an option for "revision-notify" for this rubric as there is none on the website.

gg-sys1 replied:
improved on this

doamatto commented on products/zoho.toml:
```diff
@@ -0,0 +1,77 @@
+[rubric.data-deletion]
+value = "yes-contact"
+notes = [
+  "The policy mentions that users can request deletion of their service data (e.g., \"request export or deletion of your service data\") and specifies a clean-up cycle for data removal from active databases (6 months)."
```
This citation doesn't support `yes-contact`. It needs to be more akin to saying "this is how you reach out to us to delete your data" or "this is the self-service deletion process you can use".

gg-sys1 replied:
added the right information

doamatto commented on products/zoho.toml:
```diff
@@ -0,0 +1,77 @@
+[rubric.data-collection-reasoning]
+value = "yes"
+notes = [
+  "The policy explicitly describes the purposes of data collection, such as using personal data for service delivery, communication, and customer support (e.g., generating invoices, sending marketing emails, etc.)."
```
Could you add citations for the reasoning ?

gg-sys1 replied:
added citations

doamatto commented on products/zoho.toml:
```diff
@@ -0,0 +1,77 @@
+
+[rubric.behavioral-marketing]
+value = "yes-opt-out"
+citations = ["Part I: \"Opt out of non-essential electronic communications... by using the ‘unsubscribe’ function.\""]
```
This seems to be just e-mails — what about the rest of the marketing/advertising and analytics ? If there's no opt-out, this should be changed to `yes`.

gg-sys1 replied:
updated citation and modified the value.

doamatto reviewed (changes requested):
Some notes to get started. I'll do a deeper look against Zoho's policy sometime tomorrow, if not by the end of the week.

Anon-sec commented:
> Some notes to get started. I'll do a deeper look against Zoho's policy sometime tomorrow, if not by the end of the week.

Thanks in advance for checking the policy. I'll get these changes done as soon as I'll get a little time.

Anon-sec commented:
> > Some notes to get started. I'll do a deeper look against Zoho's policy sometime tomorrow, if not by the end of the week.
> 
> Thanks in advance for checking the policy. I'll get these changes done as soon as I'll get a little time.

If you added this website before me, I'd be grateful.

doamatto commented:
Either of us can make an issue to add Zoho — I eventually plan to get to more of the issues relating to adding new policies. You're more than welcome to keep at it, if you want

Anon-sec commented:
> Either of us can make an issue to add Zoho — I eventually plan to get to more of the issues relating to adding new policies. You're more than welcome to keep at it, if you want

Thanks I'll keep working on this one then.

Anon-sec committed:
update zoho.toml and added new zoho square icon

github-actions[bot] committed:
chore: update Browserslist db

Anon-sec committed:
fixed rubric error

Anon-sec committed:
improved icon

Anon-sec committed:
Edited rubric for behavioral-marketing

Anon-sec committed:
improved icon

Anon-sec committed:
resized image and fixed background

Anon-sec committed:
Merge pull request #1 from Anon-sec/browserslist-update

Update Browserslist database

Anon-sec committed:
n/a

Anon-sec committed:
Merge branch 'master' of https://github.com/Anon-sec/privacyspy

doamatto commented on package-lock.json:
I'd rather this and the `package.json` get reverted — I don't have the time right now to try and figure the package changes right this moment. `jest` is slated to be updated in #171 and efforts can be made there if you want

gg-sys1 replied:
reverted them back

doamatto commented on package.json:
```diff
@@ -42,7 +42,7 @@
     "image-size": "^0.8.3",
     "is-core-module": "^2.4.0",
-    "jest": "^27",
+    "jest": "^27.5.1",
```
See comment on `package-lock.json`

doamatto commented on products/zoho.toml:
```diff
@@ -0,0 +1,109 @@
+name = "Zoho Corp"
+description = "Zoho Corp is a global software company that offers a suite of cloud-based applications for business productivity, including CRM, project management, and marketing automation tools."
```
I think I'd leave out the bits after « business productivity » just for brevity

gg-sys1 replied:
ok

doamatto committed:
fix(product): various Zoho changes

- Remove some extra notes
- Add back some typos (preserve Zoho's word)
- Removed some extra text in some citations
- Added modified date
- Converted some citations into notes (not quotes)
- Added more detailed information to `rubric.security`

doamatto commented on products/zoho.toml:
```diff
@@ -0,0 +1,109 @@
+[rubric.third-party-collection]
+value = "yes"
+citations = [
+  "Federated Authentication (e.g., Google, LinkedIn): Name, Email, Referrals: Name, Email, Partners & Service Providers: Name, Email, Company, Social Media/Public Sources: Profile info (name, photo, posts)."
```
Should be a note since it's not a quote. Breaking this up could help things a wee bit as well

gg-sys1 replied:
changed it as per your suggestion.

doamatto reviewed (changes requested):
Few more things to do — made some of the changes myself, but here's what's left to be taken care of. Thanks for the work so far !

Anon-sec committed:
restored package.json

Anon-sec committed:
Merge branch 'master' of https://github.com/Anon-sec/privacyspy

Anon-sec committed:
made the requested changes.

Anon-sec commented:
@doamatto made changes as requested :)

doamatto committed:
fix(product): amend some quotes that weren't searchable

doamatto reviewed (approved):
Sorry for the delay. Things look good so I'll merge when the checks pass.
